docs/conf.py

```diff
@@ -184,6 +184,10 @@
     'css_files': ['_static/css/tabs.css'],
 }
 
+development = True if release.startswith('3.0') else False
```

Comment:
I have a question here. We need to figure out how we will distinguish development version. (I hope that) 3.0 won't be forever development version.